mapper-pojo/src/main/java/org/hibernate/search/v6poc/entity/pojo/bridge/Bridge.java

```diff
@@ -6,22 +6,64 @@
  */
 package org.hibernate.search.v6poc.entity.pojo.bridge;
 
-import org.hibernate.search.v6poc.backend.document.model.IndexSchemaElement;
 import org.hibernate.search.v6poc.backend.document.DocumentState;
+import org.hibernate.search.v6poc.backend.document.model.IndexSchemaElement;
 import org.hibernate.search.v6poc.entity.model.SearchModel;
 import org.hibernate.search.v6poc.entity.pojo.model.PojoModelElement;
 import org.hibernate.search.v6poc.entity.pojo.model.PojoState;
 
 /**
+ * A bridge between an element of the POJO model and an element of the index schema.
+ * <p>
+ * The {@link Bridge} interface is a more powerful version of {@link FunctionBridge},
+ * in which a property can be mapped to more than one field, in particular.
+ *
  * @author Yoann Rodiere
  */
 public interface Bridge extends AutoCloseable {
```

Comment:
Didn't we talk about renaming this one to `PojoBridge`?

Reply:
It was about splitting it into two classes, `PojoBridge` (or `PojoClassBridge`, I don't remember) and `PojoPropertyBridge`, if I remember correctly. Yes, we talked about it, but I still have to work on it.

Reply:
OK, remembered the `Pojo` part :). Let's do it later then.